### PR TITLE
[IOS-1783] Added configurations, tweaked the styling logic and output slightly

### PR DESCRIPTION
**Pull request by gpkash**

### CODE REVIEW

**Comment by gpkash** [comment]
@lymburn I'm not adding documentation and tests just yet since this is likely to change a bit before it going live.

File: Sources/Markdowntown/Markdowntown.swift [hunk 1]
```diff
@@ -41,52 +37,89 @@ public struct Markdowntown {
...
-            
-            let result = NSMutableAttributedString(string: "\u{00A0} \u{0009} \u{00A0}\(newLines)")
+            let result: NSMutableAttributedString
             
-            stylesheet.applyStyling(thematicBreak: result)
+            if configuration.useThematicBreak {
+                result = NSMutableAttributedString(string: "\u{00A0} \u{0009} \u{00A0}")
+                
+                if thematicBreak.hasSuccessor {
+                    result.append(NSAttributedString(string: "\n"))
```
Comment (lymburn) [inline Sources/Markdowntown/Markdowntown.swift:77]: Why is this only using 1 `\n` unlike before with 2?

**Review by lymburn — APPROVED** [review]
Overall looks good!

File: Sources/Markdowntown/Markdowntown.swift (hunk 1 above)
Comment (gpkash) [reply Sources/Markdowntown/Markdowntown.swift:77]: Thanks for the reminder - I _might_ revert this but I'll leave it like this for now.

**Pull request merged**